ONSdigital/response-operations-ui
# Upgrade  to pattern library

# Motivation and Context
Upgrade the response-ops repo to make proper use of the pattern library, and extract unnecessary code out.

# What has changed
* CSS removed that is also present in the pattern library
* A number of templates refactored to use pattern library markup
* A number of social code files, and their tests, removed to avoid fixing unit tests on unused code.

# How to test?
* Ensure the response-ops repo will run
* Ensure the response-ops repo units tests pass
* Run through user journeys to make sure that no UI regressions have occurred, and no journeys now fail.

# Links
https://trello.com/c/70JT2dJk/468-bring-in-pattern-library-to-response-ops-m
